internal/template/templates/.claude/skills/moai-domain-uiux/modules/design-system-tokens.md

```
@@ -0,0 +1,441 @@
+writeFileSync('src/tokens/colors.ts', types);
+```
+
+### Visual Regression Testing (Chromatic)
+
+```bash
+# Install Chromatic
+npm install --save-dev chromatic
+```
+
+CI/CD Integration:
+```yaml
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)